packages/gitbook-docs/src/markdoc/schema/code.ts

```diff
@@ -27,7 +28,7 @@ export const code: Schema = {
   attributes: {
     title: { type: String },
     overflow: { type: String },
-    lineNumbers: { type: BooleanAttr },
+    lineNumbers: { type: BooleanOrNullAttr },
```

Comment:
What about to add a test? The [test block is already there](https://github.com/pagopa/developer-portal/pull/307/files#diff-1f878dc19510fc231071b13b3bb428deeecc7b828ba093f1197592a54f3f3f8bR315-R350)

Reply:
Done